src/main/java/io/vlingo/symbio/store/object/jdbc/JDBCObjectStoreActor.java

```diff
@@ -173,7 +174,8 @@ public void queryObject(final QueryExpression expression, final QueryResultInter
    */
   @Override
   public void registerMapper(final PersistentObjectMapper mapper) {
-    delegate.registerMapper(mapper);
+    this.delegate.registerMapper(mapper);
+    this.dispatchControlDelegate.registerMapper(mapper);
```

Comment:
You should instead send a message to the `DispatcherControl` because you are modifying state that should be owned exclusively by another actor.